Tests/SwiftStdlibTests/BinaryIntegerExtensionsTests.swift

```diff
@@ -4,6 +4,20 @@
 import XCTest
 
 final class BinaryIntegerExtensionsTests: XCTestCase {
+    func testBytes() {
+        let zero = Int32.zero.bytes
+        assert(zero == Array(repeating: 0, count: 32 / 8))
```

Comment:
Please fix all the tests here:
1. Use `XCTAssertEqual` instead of `assert`.
2. Explicitly write the count expected, no need to write it as a division.

```suggestion
        XCTAssertEqual(zero, Array(repeating: 0, count: 4))
```